tests/unit/api/objectPutRetention.js

```diff
@@ -95,8 +101,15 @@ const putObjRetRequestGovernanceLonger = {
     post: objectRetentionXmlGovernanceLonger,
 };
 
+const putObjRetRequestGovernanceShorter = {
+    bucketName,
+    objectKey: objectName,
+    headers: { host: `${bucketName}.s3.amazonaws.com` },
+    post: objectRetentionXmlGovernanceShorter,
+};
+
 const expectedMode = 'GOVERNANCE';
-const expectedDate = date.toISOString();
+const expectedDate = moment().add(2, 'days').toISOString();
```

Comment:
isn't there a theoretical flakiness risk if there's millisec skew (or overlap) while running the test, making the dates different just because the 2 values compared are evaluated a 2 different call sites? Not a big deal, but could be painful to track down, idk.